inout.py

```diff
@@ -63,6 +63,14 @@ def fileDictHeaderLists(file, key, val, delim="\t"):
                 fileD[cols[headD[key]]].append(cols[headD[val]])
     return fileD
 
+def fileDictLists(file, key=0, val=1, delim="\t"):
```

Comment:
Still missing the 'header' argument. While I acknowledge that a user is more likely to use fileDictHeaders if they have a header row, adding this argument makes this function more compatible with fileDict and allows users to still specify column indices, even if there is a header row. 

If header=True, then you just need to skip the first line. If header=False, then you don't skip the first line.